scripts/notes.ts

```diff
@@ -37,7 +37,8 @@ const rl = readline.createInterface({
 /**
  * Current milestone reference
  */
-const cm = JSON.parse(execSync('gh api repos/CMSgov/design-system/milestones').toString())[0];
+const milestone = JSON.parse(execSync('gh api repos/CMSgov/design-system/milestones').toString())[0];
+const { title: milestoneTitle, open_issues, closed_issues } = milestone;
```

Comment:
This changed but references to it didn't, so the script throws errors now

Reply:
Yes still working on it! didn't push my changes yet.